install_config/router/default_haproxy_router.adoc

```diff
@@ -1491,22 +1491,33 @@ for consumption by external metrics collection and aggregation systems (e.g. Pro
 Metrics are also available dirctly from the
 link:https://cbonte.github.io/haproxy-dconv/1.5/configuration.html#9[HAProxy router] in its own CSV format.
 
-When you create a router, as below,
+When you create a router using the following command, metrics are automatically available in Prometheus format on the stats-port, default 1936. 
+
 ====
 ----
 $ oc adm router --service-account=router
 ----
 ====
-metrics are automatically available in Prometheus format on the stats-port, default 1936. To suppress metrics collection,
+
+* To get the Prometheus statistics for the exposed metrics:
++
+Launch the stats window use the following URL in a browser:
++
+----
+http://admin:<Password>@<router_IP>:1936/metrics
+----
++
+To suppress metrics collection,
++
 ====
 ----
 $ oc adm router --service-account=router --stats-port=0
 ----
 ====
 
-To switch to the HAProxy CSV format metrics, edit the xref:../../architecture/networking/routes.adoc#env-variables[environment variables]
+* To switch to the HAProxy CSV format metrics, edit the xref:../../architecture/networking/routes.adoc#env-variables[environment variables]
```

Comment:
Above should be `switch to the HTML format metrics` instead of `CSV format`, see https://bugzilla.redhat.com/show_bug.cgi?id=1548500